packages/storybook/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -557,6 +557,7 @@ const MessageThreadStory = (args): JSX.Element => {
             />
           );
         }}
+        enableRichTextEditor={args.enableRichTextEditor}
```

Comment:
It looks like the message needs to be updated to html type after editing it in RTE or all html tags are shown. Do you know if we can update this?

Reply:
Updated